### PR TITLE
Matthew area graph

### PR DESCRIPTION
**Pull request by MatthewHiscock**
@mariahhamel, @chriddyp, @NicoleGrondin, @aneda, @mkcor - standard image renaming and a few other minor changes

### CODE REVIEW

**Comment by mkcor** [comment]
Where is / is there a reference in our (updated?) style guide for using boldface on elements of the Web App (e.g., **TRACES**)?

**Comment by MatthewHiscock** [comment]
Hey Marianne,

I’ve been using this non-final draft doc:

https://docs.google.com/document/d/1xSiXgqLPzGPXDa9EpTaOqvuHXqTDf4Ky_0wXOSKxN7Q/edit

TBH I haven’t been super-strict with sticking to it. Tho once I get the hang of GitHub I’ll take a quick sweep through what I’ve done so far and make it all more uniform.

**Comment by mkcor** [comment]
Amazing, @MatthewHiscock , that would be the reference by default, at least for now.

But then, if you make a new decision, such as styling Web App elements, please add this new rule in the style guide, so we're all on the same page.

**Comment by MatthewHiscock** [comment]
Will do.

File: _posts/2015-04-21-How-to-make-an-area-graph.md [hunk 1]
```diff
@@ -23,21 +23,21 @@ Area plots are the same as scatter and line plots, but with different trace opti
 ###Step 1: Setting up the data in the grid
 
 
-This is how you would set up Plotly’s grid to make the graph above (data here: [https://plot.ly/~Dreamshot/566](https://plot.ly/~Dreamshot/566])). Click the blue plot button to create the chart. | ![How to make an area chart](/static/images/How-to-make-an-area-graph/image10.png)
+This is how you would set up Plotly’s grid to make the graph above (You can grab the data [here](https://plot.ly/~Dreamshot/566)). Select **choose as X** for Column 1 and select **choose as Y** for both Columns 2 and 3, then click the blue plot button to create the chart. | ![LA vs NY population](/static/images/How-to-make-an-area-graph/la-vs-ny-population.png)
```
Comment (chriddyp) [inline _posts/2015-04-21-How-to-make-an-area-graph.md]: You -> you, right?

**Comment by chriddyp** [comment]
otherwise looks good! :dancer:

File: _posts/2015-04-21-How-to-make-an-area-graph.md (hunk 1 above)
Comment (MatthewHiscock) [inline _posts/2015-04-21-How-to-make-an-area-graph.md]: I don't quite follow what you mean.... 

The prior version didn't quite work without that extra step, when I followed it.

Comment (MatthewHiscock) [inline _posts/2015-04-21-How-to-make-an-area-graph.md]: Maybe I'm looking at the wrong thing? @chriddyp

Comment (chriddyp) [inline _posts/2015-04-21-How-to-make-an-area-graph.md]: Oh, I just mean uncapitalize "You"

Comment (MatthewHiscock) [inline _posts/2015-04-21-How-to-make-an-area-graph.md]: ah, cool - gotcha! Yes, you're right.

**Pull request merged**